app/components/Views/LedgerConnect/index.tsx

```diff
@@ -0,0 +1,288 @@
+            ledgerDeviceDarkImage,
+          )}
+          style={styles.ledgerImage}
+        />
+        <Text bold style={styles.connectLedgerText}>
+          {strings('ledger.connect_ledger')}
+        </Text>
+        <View style={styles.imageContainer}>
+          <Image
+            source={useAssetFromTheme(
+              ledgerConnectLightImage,
+              ledgerConnectDarkImage,
+            )}
+            style={styles.coverImage}
+          />
+        </View>
+        <View style={styles.textContainer}>
+          <View style={styles.lookingForDeviceContainer}>
+            <Text style={styles.lookingForDeviceText} bold>
+              {!isAppLaunchConfirmationNeeded
+                ? strings('ledger.looking_for_device')
+                : strings('ledger.open_eth_app')}
+            </Text>
+            {!selectedDevice && (
+              <ActivityIndicator style={styles.activityIndicatorStyle} />
+            )}
+          </View>
+          {!isAppLaunchConfirmationNeeded ? (
+            <>
+              <Text style={styles.instructionsText}>
+                {strings('ledger.ledger_reminder_message')}
+              </Text>
+              <Text style={styles.ledgerInstructionText}>
+                {strings('ledger.ledger_reminder_message_step_one')}
+              </Text>
+              <Text style={styles.ledgerInstructionText}>
+                {strings('ledger.ledger_reminder_message_step_two')}
+              </Text>
+              <Text style={styles.ledgerInstructionText}>
+                {strings('ledger.ledger_reminder_message_step_three')}
+              </Text>
+              <Text style={styles.howToInstallEthAppText} bold big link>
+                {strings('ledger.how_to_install_eth_app')}
+              </Text>
+            </>
+          ) : (
+            <Text>
+              {strings('ledger.open_eth_app_message_one')}{' '}
+              <Text bold>{strings('ledger.open_eth_app_message_two')} </Text>
+            </Text>
+          )}
+        </View>
+        <View style={styles.bodyContainer}>
+          {!isAppLaunchConfirmationNeeded && (
+            <Scan
+              onDeviceSelected={(ledgerDevice) =>
+                setSelectedDevice(ledgerDevice)
+              }
+            />
+          )}
+
+          {selectedDevice && !isAppLaunchConfirmationNeeded && (
+            <View style={styles.buttonContainer}>
+              <StyledButton
+                type="confirm"
+                onPress={() => connectLedger(true)}
+                testID={'add-network-button'}
+                disabled={isSendingLedgerCommands}
+              >
+                {isSendingLedgerCommands ? (
+                  <ActivityIndicator color={colors.white} />
+                ) : ledgerError ? (
+                  strings('ledger.retry')
+                ) : (
+                  strings('ledger.continue')
+                )}
+              </StyledButton>
+            </View>
+          )}
```

Comment:
Nit: Prefer ternary operator

```suggestion
          {(selectedDevice && !isAppLaunchConfirmationNeeded) ? (
            <View style={styles.buttonContainer}>
              <StyledButton
                type="confirm"
                onPress={() => connectLedger(true)}
                testID={'add-network-button'}
                disabled={isSendingLedgerCommands}
              >
                {isSendingLedgerCommands ? (
                  <ActivityIndicator color={colors.white} />
                ) : ledgerError ? (
                  strings('ledger.retry')
                ) : (
                  strings('ledger.continue')
                )}
              </StyledButton>
            </View>
          ) : null}
```